controller/app/cmdline/src/cmd_line.cpp

```diff
@@ -1409,6 +1420,42 @@ int cmd_line::cmd_view_all(int total_matched, std::vector<cli_argument*> args)
     return 0;
 }
 
+int cmd_line::cmd_view_stream_formats(int total_matched, std::vector<cli_argument*> args)
+{
+    struct format_and_description
+    {
+        unsigned int stream_format_index;
+        const char *str;
+        const char *description;
+    };
+    
+    struct format_and_description stream_formats[] =
+    {
+        {0, "IEC...48KHZ_1CH", "IEC61883_AM824_MBLA_48KHZ_1CH"},
+        {1, "IEC...48KHZ_2CH", "IEC61883_AM824_MBLA_48KHZ_2CH"},
+        {2, "IEC...48KHZ_4CH", "IEC61883_AM824_MBLA_48KHZ_4CH"},
+        {3, "IEC...48KHZ_8CH", "IEC61883_AM824_MBLA_48KHZ_8CH"},
+        {4, "IEC...96KHZ_1CH", "IEC61883_AM824_MBLA_96KHZ_1CH"},
+        {5, "IEC...96KHZ_2CH", "IEC61883_AM824_MBLA_96KHZ_2CH"},
+        {6, "IEC...96KHZ_4CH", "IEC61883_AM824_MBLA_96KHZ_4CH"},
+        {7, "IEC...96KHZ_8CH", "IEC61883_AM824_MBLA_96KHZ_8CH"},
+        {8, "UNKNOWN", "UNKNOWN"}
+    };
```

Comment:
These shouldn't be in the cmdline application.......
Can't we read them to util.cpp somehow?